query/src/main/kotlin/com/linecorp/kotlinjdsl/query/spec/expression/ColumnSpec.kt

```diff
@@ -33,4 +34,9 @@ data class ColumnSpec<T>(
 
     private fun path(froms: Froms): Path<T> =
         froms[entity].get(path)
+
+    fun <NT> nested(property: KProperty1<T, NT>): NestedColumnSpec<NT> = NestedColumnSpec(
+        this,
+        property.name
+    )
```

Comment:
This looks like the method used in the DSL layer. Therefore, it would be better to declare it as an extension function in DSL.

Reply:
thank you I changed